com.openwall.John.json

```diff
@@ -40,14 +43,13 @@
             "buildsystem": "simple",
             "subdir": "src",
             "build-commands": [
-                "###### build script ######",
-                "../build.sh"
+                "./configure && make"
```

Comment:
Doing so will remove the SIMD chain/fallback. Since this is a package that will run on a multitude of different machines, this is a bad thing. You are depriving users of getting the maximum speed (if you build on an old CPU) or you will make it impossible for users with an old CPU to use john (if you build on an updated CPU).

Reply:
You are replacing 10 different binaries with just one. There is a reason to build and package 10 different binaries.